hightouch_blueprints/verify_sync.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+import errors
```

Comment:
Same comment on relative import if possible.

Reply:
And same comment as I made on the other PR. :)